php-frontend/src/main/java/org/sonar/php/utils/ExpectedIssuesParser.java

```diff
@@ -20,19 +20,21 @@
 package org.sonar.php.utils;
 
 import com.google.common.base.Splitter;
-import java.io.File;
 import java.util.ArrayList;
 import java.util.List;
+
+import org.sonar.php.compat.CompatibleInputFile;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.sonar.php.compat.CompatibleInputFile'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)